src/handlers/wyre.js

```diff
@@ -126,9 +134,12 @@ export const trackWyreOrder = async (referenceInfo, orderId) => {
   }
 };
 
-export const trackWyreTransfer = async (referenceInfo, transferId) => {
+export const trackWyreTransfer = async (referenceInfo, transferId, network) => {
   try {
-    const response = await wyreApi.get(`/v2/transfer/${transferId}/track`);
+    const baseUrl = getBaseUrl(network);
+    const response = await wyreApi.get(
+      `${baseUrl}/v2/transfer/${transferId}/track`
```

Comment:
out of curiosity, why do we hit 2 different versions of their API for different endpoints? (v2 & v3)

Reply:
Yes, this is according to their docs. Double checked recently to see if anything had been updated on their docs for the v2, but it's still the same